### PR TITLE
Add JSON Rendering

### PR DESCRIPTION
**Pull request by DakshMiglani**
# Added JSON Rendering.

for issue: #961

Dependencies added: `json-formatter-js`

Possible Use Cases? : 

1. Beautifies the json
2. Helps developers understand json
3. ...addMore

### CODE REVIEW

**Comment by DakshMiglani** [comment]
@pfrazee can you check this?

**Comment by taravancil** [comment]
Hi @DakshMiglani! We saw this but haven't reviewed it yet! Be patient - we have 100 or so other things on our TODO list :smile:

**Comment by DakshMiglani** [comment]
@taravancil Oh, alright. paul didn't reply so i thought you guys have not got the time. It's all cool tho.

File: app/json-renderer.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import createJSON from './lib/fg/json'
+
+const isJSON = function (str) {
+  if (str === "" || str === ' ') return false
+  str = str.replace(/\\(?:["\\\/bfnrt]|u[0-9a-fA-F]{4})/g, '@')
+  str = str.replace(/"[^"\\\n\r]*"|true|false|null|-?\d+(?:\.\d*)?(?:[eE][+\-]?\d+)?/g, ']')
+  str = str.replace(/(?:^|:|,)(?:\s*\[)+/g, '')
+  return (/^[\],:{}\s]*$/).test(str)
```
Comment (pfrazee) [inline app/json-renderer.js]: Would it be simpler to just do `JSON.parse(str)` and return false if it throws?

Comment (DakshMiglani) [reply app/json-renderer.js]: let's say there's a very large JSON string, parsing it to the memory might even cause the browser to crash just to know that it's json, instead just use a regular expression to understand if it is of type json or not.

Comment (pfrazee) [reply app/json-renderer.js]: That's true. In that case, I'd suggest we check the string length first, and return false if it's >2mb. It'd make sense not to handle huge JSONs anyway, right?

Comment (DakshMiglani) [reply app/json-renderer.js]: yea that can be done.

Comment (DakshMiglani) [reply app/json-renderer.js]: @pfrazee can you put the exact bounds for the max length of the string, i can submit a few commits.

**Comment by pfrazee** [comment]
Merged! Thanks for the contribution!

**Comment by pfrazee** [comment]
We ended up changing the theme to be light, and adding a raw/formatted toggle interface

**Comment by DakshMiglani** [comment]
Oh, thanks!

**Pull request merged**